libs/foundry-wrapper/src/utils/executable.rs

```diff
@@ -0,0 +1,20 @@
+use crate::Error;
+use std::path::PathBuf;
+
+pub fn find_forge_executable() -> Result<PathBuf, Error> {
+    which::which("forge").map_err(|_| Error::FoundryExecutableNotFound)
```

Comment:
Should I do to same with slither and solc in the slither impl @0xMemoryGrinder ?